4-esm-webpack/package.json

```diff
@@ -2,23 +2,23 @@
   "name": "npm-es-modules-4-esm-webpack",
   "version": "0.0.2",
   "description": "Breakdown of 7 different ways to use ES modules with NPM today.",
-  "main": "main.js",
-  "module": "module.jsm",
+  "main": "index.js",
+  "module": "main.js",
   "browser": "dist/browser.js",
   "repository": "transitive-bullshit/npm-es-modules",
   "author": "Travis Fischer <travis@automagical.ai>",
   "license": "MIT",
   "scripts": {
     "prebuild": "yarn run clean",
-    "build": "webpack",
-    "start": "webpack -w",
+    "build": "webpack -r esm",
+    "start": "webpack -r esm -w",
     "clean": "del dist",
```

Comment:
👆 `esm` can be used to load `webpack` configs too

Reply:
<3